File: src/main/java/com/zhongan/devpilot/completions/inline/DefaultCompletionAdjustment.java
//
// Source code recreated from a .class file by IntelliJ IDEA
// (powered by FernFlower decompiler)
//

package com.zhongan.devpilot.completions.inline;

import com.zhongan.devpilot.completions.general.SuggestionTrigger;

public class DefaultCompletionAdjustment extends CompletionAdjustment {
    public DefaultCompletionAdjustment() {
    }

    public SuggestionTrigger getSuggestionTrigger() {
        return SuggestionTrigger.DocumentChanged;
    }
}

File: src/main/java/com/zhongan/devpilot/completions/inline/DefaultCompletionAdjustment.kt
package com.zhongan.devpilot.completions.inline

import com.zhongan.devpilot.completions.general.SuggestionTrigger

class DefaultCompletionAdjustment : CompletionAdjustment() {

    override val suggestionTrigger: SuggestionTrigger
        get() = SuggestionTrigger.DocumentChanged
}
